Debian Bug#196012: FTBFS on hppa

diff -Ndur mikmod-3.1.6.orig/src/mconfedit.c mikmod-3.1.6/src/mconfedit.c
--- mikmod-3.1.6.orig/src/mconfedit.c	2003-06-04 11:59:01.000000000 +0200
+++ mikmod-3.1.6/src/mconfedit.c	2003-06-04 12:02:45.000000000 +0200
@@ -164,7 +164,7 @@
 	if (entry->help) free(entry->help);
 	va_start(args,str);
 	vsprintf(storage,str,args);
-	va_end(str);
+	va_end(args);
 
 	len=MIN(strlen(storage),STORAGELEN);
 	entry->help=malloc(sizeof(char)*(len+1));
